diff --git a/proxy/examples/gssproxy.conf.in b/proxy/examples/gssproxy.conf.in
index 321a4e7..5dd708b 100644
--- a/proxy/examples/gssproxy.conf.in
+++ b/proxy/examples/gssproxy.conf.in
@@ -1,5 +1,11 @@
 [gssproxy]

+[service/HTTP]
+  mechs = krb5
+  cred_store = keytab:/etc/gssproxy/http.keytab
+  cred_store = ccache:/var/lib/gssproxy/clients/krb5cc_%U
+  euid = 48
+
 [service/nfs-server]
   mechs = krb5
   socket = /run/gssproxy.sock
