diff --git a/proxy/systemd/gssproxy.service.in b/systemd/gssproxy.service.in
index 7aa6785..dae39ee 100644
--- a/proxy/systemd/gssproxy.service.in
+++ b/proxy/systemd/gssproxy.service.in
@@ -6,6 +6,7 @@ Before=nfs-secure.service nfs-secure-server.service
 Requires=proc-fs-nfsd.mount

 [Service]
+Environment=KRB5RCACHEDIR=/var/lib/gssproxy/rcache
 ExecStart=@sbindir@/gssproxy -D
 # These two should be used with traditional UNIX forking daemons
 # consult systemd.service(5) for more details
